File: example/src/androidTest/java/eu/darken/mvpbakery/example/ExampleApplicationMock.java
package eu.darken.mvpbakery.example;

import android.app.Activity;

import eu.darken.mvpbakery.injection.ManualInjector;

public class ExampleApplicationMock extends ExampleApplication {

    public void setActivityComponentSource(ManualInjector<Activity> injector) {
        this.manualInjector = injector;
    }
}

File: example/src/androidTest/java/eu/darken/mvpbakery/example/ExampleApplicationMock.kt
package eu.darken.mvpbakery.example

import android.app.Activity

import eu.darken.mvpbakery.injection.ManualInjector

class ExampleApplicationMock : ExampleApplication() {

    fun setActivityComponentSource(injector: ManualInjector<Activity>) {
        this.manualInjector = injector
    }
}
